lib/model/autocomplete.dart

```diff
@@ -939,10 +943,16 @@ class TopicAutocompleteView extends AutocompleteView<TopicAutocompleteQuery, Top
 class TopicAutocompleteQuery extends AutocompleteQuery {
   TopicAutocompleteQuery(super.raw);
 
-  bool testTopic(TopicName topic) {
+  bool testTopic(PerAccountStore store, TopicName topic) {
     // TODO(#881): Sort by match relevance, like web does.
+
+    if (topic.displayName == null) {
+      return store.realmEmptyTopicDisplayName.toLowerCase()
+        .contains(raw.toLowerCase());
+    }
+    // Skip this option if it matches the query exactly.
     return topic.displayName != raw
```

Comment:
This comment seems to just restate what the line after it says. Cleanest to leave it out.